src/_locales/en/messages.json

```
@@ -284,5 +292,13 @@
     "plugin_file_service_drop_here": "Drop a file here to upload",
     "plugin_file_service_error_101": "The input is not a single file.",
     "plugin_file_service_error_102": "The file is too large; limit is {{limit}}.",
-    "plugin_file_service_on_change_file": "Change File"
+    "plugin_file_service_on_change_file": "Change File",
+    "plugin_trader_no_data": "No Data",
+    "plugin_trader_tab_price": "Price",
+    "plugin_trader_tab_exchange": "Price",
+    "plugin_trader_tab_switch_data_source": "Switch Data Source:",
+    "plugin_trader_table_exchange": "Exchange",
+    "plugin_trader_table_pair": "Pair",
```

```suggestion
    "plugin_trader_table_pair": "Currency pair",
```

see https://en.wikipedia.org/wiki/Currency_pair

---

It's from CoinGecko. see https://www.coingecko.com/en/widgets/coin_market_ticker_list_widget